source/chapters/library.rst

```diff
@@ -866,8 +871,42 @@ different location. Select :guilabel:`Use Default Library` to reset.
   Mac partition mounted in Linux, you can load it and use your iTunes tracks and
   playlists as well.
 
-.. seealso:: External libraries can be disabled under
-             :menuselection:`Preferences --> Library`.
+.. _library-serato:
+
+Using the Serato library
+------------------------
+
+Mixxx supports reading your Serato library and crates both from your local hard drive as well as portable USB drives. Smart crates are currently not supported.
+
+Serato stores information such as hot cues, track color and beatgrid in the file tags.
+Regardless of whether a track is loaded into deck from the libray or directly from the file browser, Mixxx will import that information automatically if present.
+
+Saved loops are imported as well, but since Mixxx is lacking support for saved loops, only the first loop is usable in Mixxx.
```

Comment:
Maybe:

Saved loops are imported as well, but since Mixxx is lacking support for mulitple saved loops, only the first loop is usable in Mixxx.

(add the word "multiple")

Reply:
Changes effected @ywwg! Thank you for the reviews @Be-ing, @Holzhaus, @ronso0 and @ywwg.